src/reducers/PrescriptionReducer.js

```diff
@@ -10,7 +10,7 @@ import { PRESCRIPTION_ACTIONS } from '../actions/PrescriptionActions';
 const initialState = () => ({
   currentTab: 0,
   transaction: null,
-  items: UIDatabase.objects('Item'),
+  items: UIDatabase.objects('Item').filtered('isVaccine != true'),
```

Comment:
So we won't allow dispensing of vaccines in a 'normal' prescription?
I don't remember this coming up as a requirement, but I think its a good idea

Reply:
Was mainly because the spinner can't handle decimals without tweaks - I thought this was easier than tweaks .. heh